### ISSUE TITLE
Coordinates of the joints

### ISSUE THREAD

**Issue by akhilgupta0221** [TianzhongSong/Real-Time-Action-Recognition#12]
Hi

Can you please tell me how to fetch the coordinates of each joint especially the coordinates of palms of our hands.

Regards
Akhil

**Comment by TianzhongSong** [comment]
@akhilgupta0221 There are 18 joints of a person. 
https://github.com/TianzhongSong/Real-Time-Action-Recognition/blob/master/pose/common.py#L13

**Comment by akhilgupta0221** [comment]
@TianzhongSong : Yes that's ok  but I would like to dump all the coordinates ok 18 joiints in a live stream
Where to fetch the coordinates ???

I tried printing the center coordinates at @line: 335 from the function  get_humans() in the file estimator.py.

Are those the correct coordinates

**Comment by TianzhongSong** [comment]
@akhilgupta0221 Yes, they are.

**Comment by akhilgupta0221** [comment]
@TianzhongSong : Thanks !!!!
